docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -0,0 +1,257 @@
+======================
+Trino client REST API
+======================
+
+The Trino client allows users to submit Trino queries and view results.  This
```

Comment:
This feels wrong.

How about something like this

The REST API allows clients to submit SQL queries to Trino and receive the results. Clients include the CLI, the JDBC, driver, and others provided by the community. The preferred method to interact with Trino is using these existing clients. This document provides details about the API for reference. It can also be used to implement your own client, if necessary.

Reply:
Very good!  Replaced.